# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1287459967 -28800
#      Tue Oct 19 11:46:07 2010 +0800
# Node ID f7079e3b0227b4d47d4762092136eaa36a15b69b
# Parent  a8ac114ec9ab5f1de499b9e70244446f08261cfb
strjust.m: Add "untabify" and "strrep" to @seealso{} in the doc-string.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-10-19  Ben Abbott <bpabbott@mac.com>
+
+	* strings/strjust.m: Add "untabify" and "strrep" to @seealso{} in the
+	doc-string.
+
 2010-10-18  Thorsten Meyer  <thorsten.meyier@gmx.de>
 
 	* general/gradient.m, linear-algebra/normest.m,	plot/__gnuplot_print__.m,
 	plot/__print_parse_opts__.m, plot/daspect.m,
 	plot/gnuplot_drawnow.m, plot/pbaspect.m, plot/print.m,
 	signal/periodogram.m: Style fix, replace end by endfor.
 
 2010-10-18  Marco Caliari <marco.caliari@univr.it>
diff --git a/scripts/strings/strjust.m b/scripts/strings/strjust.m
--- a/scripts/strings/strjust.m
+++ b/scripts/strings/strjust.m
@@ -37,17 +37,17 @@
 ## strjust (["a"; "ab"; "abc"; "abcd"])
 ##      @result{} ans =
 ##            a
 ##           ab
 ##          abc
 ##         abcd
 ## @end group
 ## @end example
-## @seealso{deblank, strtrim}
+## @seealso{deblank, strrep, strtrim, untabify}
 ## @end deftypefn
 
 function y = strjust (x, just)
 
   if (nargin < 1 || nargin > 2)
     print_usage ();
   endif
 
